src/main/java/com/commonwealthrobotics/MainController.java

```diff
@@ -353,6 +353,16 @@ public MainController(Stage newStage) {
 		this.newStage = newStage;
 	}
 
+
```

Comment:
2 things: 

1, this maybe belongs in a different PR where this is defined in the fxml?

2: for exceptions i would like to standardize on Log.error(e); so the exception is logged